models/datastore_test.go

```diff
@@ -0,0 +1,134 @@
+package models_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/odpf/optimus/models"
+)
+
+type ResourceSpecTestSuite struct {
```

Comment:
we can avoid following the testify and use the suite functionality in go testing itself right?